src/main/java/uk/gov/hmcts/ccd/domain/service/validate/ValidateQuestionableActivity.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.ccd.domain.service.validate;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.commons.collections.MapUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.gov.hmcts.ccd.domain.model.definition.CaseDetails;
+
+import java.util.Map;
+
+public class ValidateQuestionableActivity {
+
+    private static final Logger LOG = LoggerFactory.getLogger(ValidateQuestionableActivity.class);
+    private static final String EMPTY_COLLECTION_VALUE = "\"value\":{}";
+
+    private ValidateQuestionableActivity() {
+    }
+
+    public static Boolean validateEmptyValuedCollections(CaseDetails caseDetails) {
+        if (caseDetails != null) {
+            Map<String, JsonNode> caseData = caseDetails.getData();
+            if (!MapUtils.isEmpty(caseData)) {
+                String data = jsonNodeToString(caseData);
+                if (data.contains(EMPTY_COLLECTION_VALUE)) {
```

Comment:
this doesn't look like a proper detection of empty collections we are after. It may result in lots of false positive matches. Better to traverse the object network for collection nodes in json object and more expressly check for what we are after to detect.

Reply:
Done